studio/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev -p 8082",
+    "dev:secrets:pull": "AWS_PROFILE=supabase-dev node internals/getSecrets.js",
```

Comment:
Could we get a Makefile with `make update:config` to align it with the API? Makes it easier to remember

Reply:
I think it will split the scripts into two unnecessarily (is this a npm script or a make command). We had makefile in infra cos we were coordinating across multiple systems, since studio is just one system, I think have this as a npm script is easier. And the script name aligns with the name we use if you just want to pull secrets for the API etc